react/features/invite/components/add-people-dialog/web/AddPeopleDialog.js

```diff
@@ -156,6 +157,7 @@ function AddPeopleDialog({
             <div className = 'invite-more-dialog'>
                 { _inviteContactsVisible && <InviteContactsSection /> }
                 <CopyMeetingLinkSection url = { _inviteUrl } />
+                <QrCodeSection url = { _inviteUrl } />
```

Comment:
I think the visibility of this component should be configurable, similar to the `InviteContactsSection`

Reply:
Great idea, i'll do that this evening !